blocks/api/post.pegjs

```diff
@@ -299,13 +299,13 @@ Block_Attributes
     return maybeJSON( attrs );
   }
 
-ASCII_AlphaNumeric
-  = ASCII_Letter
+ASCII_LowercaseAlphaNumeric
+  = ASCII_LowercaseLetter
```

Comment:
this seems a bit verbose. what if instead we ripped out the whole `ASCII` nomenclature and just simplified down to something like `Block_Name` or `Block_Name_Part` - we're not using the other rules so we can get rid of them

```
Namespaced_Block_Name
  = $( Block_Name_Part "/" Block_Name_Part )

Core_Block_Name
  = type:Block_Name_Part { return 'core/' + type }

Block_Name_Part
  = $( [a-z] [a-z0-9_-]* )
```

Reply:
>this seems a bit verbose. what if instead we ripped out the whole `ASCII` nomenclature and just simplified down to something like `Block_Name` or `Block_Name_Part`

Sounds good to me. Applied in df43eb4.

Reply:
looks nice!